### DISCUSSION TITLE
Problems using the bot on Windows 11

### DISCUSSION THREAD

**Discussion by Vlandmist** [sly9/VoyagerTelegramBot#46] [General]
Hi, I'm trying to install the bot for a friend, on his **Win11 PC.**

The script doesn't start, and no error msg is shown (excluding something about the localization base path).
I tried starting with VS Code (debugging) but it gets stuck deep in some astropy exception.
The script never reaches the main function in bot.py
I don't understand what the problem might be. I'm using `python version 3.9.7 (32bit)`

I installed it like this:
```
python -m pip install -r requirements.txt
python -m pip install windows-curses
```

any idea? have you ever tried running it on win11?
ty @sly9 @bigpizza

**Comment by bigpizza** [comment]
Hi Vlandmist,

We are using **VoyagerTelegramBot** with Ubuntu subsystem on Windows 11 :D The environment of Ubuntu is much more friendly to python (and other types of) scripts. If you want to have a try, please take a look at [Install Ubuntu on WSL2 and get started with graphical applications](https://ubuntu.com/tutorials/install-ubuntu-on-wsl2-on-windows-11-with-gui-support#1-overview).

**Reply by Vlandmist** [reply]
ok, I'll give it a try...! ty
